tasks.py

```diff
@@ -300,6 +349,165 @@ def post_upgrade(context):
     run_command(context, command)
 
 
+@task(
+    help={
+        "service": "Docker compose service name to run command in (default: nautobot).",
+        "command": "Command to run (default: bash).",
+        "file": "File to run command with (default: empty)",
+    },
+)
+def exec(context, service="nautobot", command="bash", file=""):
+    """Launch a command inside the running container (defaults to bash shell inside nautobot container)."""
+    command = [
+        "exec",
+        "--",
+        service,
+        command,
+        f"< '{file}'" if file else "",
+    ]
+    docker_compose(context, " ".join(command), pty=not bool(file))
+
+
+@task(
+    help={
+        "db-name": "Database name (default: Nautobot database)",
+        "input-file": "SQL file to execute and quit (default: empty, start interactive CLI)",
+        "output-file": "Ouput file, overwrite if exists (default: empty, output to stdout)",
+        "query": "SQL command to execute and quit (default: empty)",
+    }
+)
+def dbshell(context, db_name="", input_file="", output_file="", query=""):
+    """Start database CLI inside the running `db` container.
+
+    Doesn't use `nautobot-server dbshell`, using started `db` service container only.
+    """
+    if input_file and query:
+        raise ValueError("Cannot specify both, `input_file` and `query` arguments")
+    if output_file and not (input_file or query):
+        raise ValueError("`output_file` argument requires `input_file` or `query` argument")
+
+    env = {}
+    if query:
+        env["_SQL_QUERY"] = query
+
+    command = [
+        "exec",
+        "--env=_SQL_QUERY" if query else "",
+        "-- db sh -c '",
+    ]
+
+    if _is_compose_included(context, "mysql"):
+        command += [
+            "mysql",
+            "--user=$MYSQL_USER",
+            "--password=$MYSQL_PASSWORD",
+            f"--database={db_name or '$MYSQL_DATABASE'}",
+        ]
+    elif _is_compose_included(context, "postgres"):
+        command += [
+            "psql",
+            "--username=$POSTGRES_USER",
+            f"--dbname={db_name or '$POSTGRES_DB'}",
+        ]
+    else:
+        raise ValueError("Unsupported database backend.")
+
+    command += [
+        "'",
+        '<<<"$_SQL_QUERY"' if query else "",
+        f"< '{input_file}'" if input_file else "",
+        f"> '{output_file}'" if output_file else "",
+    ]
+
+    docker_compose(context, " ".join(command), env=env, pty=not (input_file or output_file or query))
+
+
+@task(
+    help={
+        "input-file": "SQL dump file to replace the existing database with. This can be generated using `invoke backup-db` (default: `dump.sql`).",
+    }
+)
+def import_db(context, input_file="dump.sql"):
```

Comment:
2 questions here:

1. Could the cookie also include a `start -- nautobot db` command? This is for cases where the containers are stopped and you run `invoke import-db`. If they aren't started, it gives the error `service "db" is not running`. It would be nice for this task to work when containers are either already started or stopped.
2. Can you test importing a dataset? I've tried a few times and it looks like it completes, but the data isn't in Nautobot

Reply:
Added auto-start of DB service.

Tested and confirming it works:

```
invoke backup-db
invoke destroy
invoke import-db
```

The only catch could be the database name, however without arguments, to commands above, it should use the default database name as specified in `development/development.env`.

You can see the database name with `head -n 30 dump.sql`:

```
...
CREATE DATABASE nautobot WITH TEMPLATE = template0 ENCODING = 'UTF8' LOCALE = 'en_US.utf8';
...
```

Reply:
@snaselj I created issue https://github.com/nautobot/cookiecutter-nautobot-app/issues/38 and PR https://github.com/nautobot/cookiecutter-nautobot-app/pull/39 addressing the issues I found with `invoke import-db` along with your solution here for running `import-db` when containers are stopped.

Reply:
Thanks @DistantVoyager , will backport useful changes from cookies to the template, after all cookies will be managed by the Drift Manager.